app/javascript/components/pages/SearchNew.js

```diff
@@ -72,322 +67,322 @@ class SearchNew extends React.Component {
     }
 
     render() {
-        return (
-            <>
-                <h3>Rank your most important category</h3>
-                <div className="body-container">
-                    <div className="form">
-                        <Form>
+      return (
+        <>
+            <h3>Rank your most important category</h3>
+            <div className="body-container">
+                <div className="form">
+                  <Form>
+                    <Row>
+                      <Col>
+                        <FormGroup>
+                            <Row>
+                                <Col>
+                                    <Label>Housing</Label>
+                                </Col>
+                                <Col>
```

Comment:
Same with this file, if you just have a predefined constant with each of these fields, you could map over them here instead of having to write them out again.

Reply:
see previous comment